### PR TITLE
Send tracking message

### PR DESCRIPTION
**Pull request by guptad2**
This pull request enables sending tracking messages when a subscribe happens

### CODE REVIEW

File: src/util/emitUtils.js [hunk 1]
```diff
@@ -576,11 +577,14 @@ function doEmit(nsp, key, obj) {
   if (!toggle.isFeatureEnabled('enableClientStats')) {
```
Comment (jgraff2) [inline src/util/emitUtils.js]: The toggle is only for the client stats - need to track emit here too.

File: src/util/emitUtils.js [hunk 2]
```diff
@@ -576,11 +577,14 @@ function doEmit(nsp, key, obj) {
...
     nsp.emit(key, newObjectAsString);
```
Comment (jgraff2) [inline src/util/emitUtils.js]: The toggle is only for the client stats - need to track emit here too.

File: src/util/emitUtils.js [hunk 3]
```diff
@@ -572,15 +573,19 @@ function emitToClients(io, nsp, rooms, key, obj) {
  */
 function doEmit(nsp, key, obj) {
   const newObjectAsString = getNewObjAsString(key, obj); // { key: {new: obj }}
+  const numClients = Object.values(nsp.connected).length;
+  tracker.trackEmit(obj.name, obj.updatedAt, numClients);
   pubSubStats.trackEmit(key, obj);
   if (!toggle.isFeatureEnabled('enableClientStats')) {
     nsp.emit(key, newObjectAsString);
   } else {
+    tracker.trackEmit(obj.name, obj.updatedAt, numClients);
```
Comment (jgraff2) [inline src/util/emitUtils.js]: Also remove from here

Comment (guptad2) [reply src/util/emitUtils.js]: fixed

File: src/util/emitUtils.js (hunk 2 above)
Comment (guptad2) [reply src/util/emitUtils.js]: fixed

**Comment by iamigo** [comment]
Were all the "error: Received invalid args in trackEmit..." being logged before?
For the two timeout errors, can you check with @jgraff2?

**Comment by guptad2** [comment]
just fixed the extra logs, will talk to john about timeouts. the, tests continuously pass on travis + local, but fail on heroku

**Comment by iamigo** [comment]
more errors now

**Comment by guptad2** [comment]
@iamigo just talked with john and tried it again, doesn't look like its happening again. Seems like it was a one of thing on heroku. @jgraff2 said not to worry about it

**Pull request merged**